### ISSUE TITLE
Doesn't work for Arduino Uno

### ISSUE THREAD

**Issue by chamik** [ad24cz/LoopSpeedLogger#1]
It just doesn't...

```last	this	min	max	avg
1999	807	-1	0	0
808	249	-1	808	808
250	73	-1	808	529
74	657	-1	808	377
658	930	-1	808	447
931	273	-1	931	544
273	544	-1	931	499
544	879	-1	931	505
879	924	-1	931	552
925	708	-1	931	593
709	441	-1	931	605
441	165	-1	931	590
165	493	-1	931	554
493	43	-1	931	550
43	987	-1	931	513
988	503	-1	988	545
504	327	-1	988	542
328	729	-1	988	530
729	840	-1	988	541
841	613	-1	988	557
613	303	-1	988	559
```

**Comment by mariansam** [comment]
Thanks! I think I have already solved the problem with `minimal()`, I'll fix `average()` asap!

**Comment by mariansam** [comment]
Should be fixed - 4788554. Does it work for you?

**Comment by chamik** [comment]
Vewy nice, thank you.
